serverless/knative_serving/serverless-service-monitoring.adoc

```diff
@@ -0,0 +1,36 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-service-monitoring"]
+= Monitoring Knative services
+include::modules/common-attributes.adoc[]
+:context: serverless-service-monitoring
+
+toc::[]
+
+You can use the {product-title} monitoring stack to record and view health checks and metrics for your Knative services. This section describes the following topics:
+
+* What metrics Knative services expose by default
+* How to configure exposing custom metrics
+* How to configure the monitoring stack to scrape the exposed metrics
+* How to view the metrics of a service
+
+[NOTE]
+====
+Scraping the metrics does not affect autoscaling of a Knative service, because scraping requests do not go through the activator. Consequently, no scraping takes place if no pods are running.
+====
+
+.Additional resources
+
+* For more information on {product-title} monitoring stack, see xref:../../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Understanding the monitoring stack].
+* For information on monitoring the components of Serverless itself, as opposed to Knative services, see xref:../../serverless/admin_guide/serverless-admin-monitoring.adoc#serverless-admin-monitoring[Monitoring serverless components].
+
+include::modules/serverless-monitoring-services-default-metrics.adoc[leveloffset=+1]
+include::modules/serverless-monitoring-services-custom-metrics.adoc[leveloffset=+1]
+
+include::modules/serverless-monitoring-services-configuration-scraping.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For more information on configuration, see xref:../../monitoring/managing-metrics.adoc#specifying-how-a-service-is-monitored[Specifying how a service is monitored].
```

Comment:
Here a minor suggestion. What about leave a reference to `Enabling monitoring for user-defined projects` section as well (Preview here https://deploy-preview-35751--osdocs.netlify.app/openshift-enterprise/latest/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects )
I was only able to get thru all these procedure after doing the procedure from this section.

Reply:
Good suggestion, done:
https://deploy-preview-35751--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_serving/serverless-service-monitoring.html#serverless-monitoring-services-configuration-scraping_serverless-service-monitoring